Src/MoneyFox.Ui/Views/Settings/SettingsViewModel.cs

```diff
@@ -31,4 +45,18 @@ public CurrencyViewModel SelectedCurrency
     }
 
     public IReadOnlyList<CurrencyViewModel> AvailableCurrencies { get; }
+
+    public AccountViewModel? SelectedAccount
+    {
+        get => selectedAccount ??= AvailableAccounts?.FirstOrDefault(x => x.Name == (string.IsNullOrEmpty(settingsFacade.DefaultAccount) ? string.Empty : settingsFacade.DefaultAccount));
+
+        set
+        {
+            SetProperty(field: ref selectedAccount, newValue: value);
+            settingsFacade.DefaultAccount = selectedAccount != null ? selectedAccount.Name : string.Empty;
+            OnPropertyChanged();
+        }
+    }
+
+    public IReadOnlyList<AccountViewModel>? AvailableAccounts => mapper.Map<ObservableCollection<AccountViewModel>>(mediator.Send(new GetAccountsQuery()).Result);
```

Comment:
If you use the `AccountPickerViewModel` you can easily map that yourself in a select. Then we can get rid of AutoMapper here :)
Also, currently there is a db call every time the getter is called. I wonder if it makes sense to make an Initialize Method that is called OnAppearing or OnNavigated to load the accounts.

Reply:
I have added a method to load the accounts when OnAppearing is called. But the properties to load the Picker are accessed even before that, so I made the getter load the accounts if it is not loaded.